dist/index.js

```diff
@@ -10292,6 +10292,10 @@ function isRC(ver) {
   return ver.match(xyzAbcVersion('^', '(?:-rc\\.?\\d+)'))
 }
 
+function isKnownBranch(ver) {
+  return ['main', 'master', 'maint'].includes(ver)
```

Comment:
If you want to be safer, you could match "maint" anywhere in the version, as OTP has maint-26, maint-25, and so on. :) Just saying, your call, we don't use it right now.